### ISSUE TITLE
Usage with standalone glimmer apps

### ISSUE THREAD

**Issue by JamesWilmot** [pzuraq/ember-could-get-used-to-this#24]
Hi @pzuraq,

I've been following this project with great interest with an especial interest in effects implementation; I think it will quite handily solve a number of pain points integrating third party libraries that we're experiencing.

Out of curiosity: would there be a way to use this in a standalone glimmerjs app? 

Thanks for hard work on this and underlying architecture!

**Comment by pzuraq** [comment]
@JamesWilmot absolutely, all of these primitives are being built into the Glimmer VM directly as they relate to the templating layer, so they'll be usable in Glimmer.js. It's actually part of the reason why Effects are taking as long as they are, rather than hacking it on top of the VM we're taking the time to do it right, and with everything else going on it's just been a bit hectic so I haven't had time to dig in there 😕  hoping to get to it early next year though!

**Comment by JamesWilmot** [comment]
Thanks heaps! Could you point me in the direction of issue/s in glimmer where work is happening? 

Cheers!

**Comment by josemarluedke** [comment]
I'm very interested in this as well. What is the state of the dependencies to make this happen?

**Comment by josemarluedke** [comment]
Just a heads up around this. I have been working on a package that is based on the Resource concept and that package is meant to be used in Glimmer and Ember. So I was able to prove out that we can actually have this working in Glimmer standalone. I had to copy the resource code into that package, but all the underlining features are available from Glimmer VM.

To follow that work, you can find the relevant PR here: https://github.com/josemarluedke/glimmer-apollo/pull/3.
